build.gradle

```diff
@@ -179,4 +179,6 @@ nexusPublishing {
 nexusStaging {
     username = System.getenv().get("SONATYPE_USER")
     password = System.getenv().get("SONATYPE_PASSWORD")
+    numberOfRetries = 120
```

Comment:
Can you explain this? Why does it have to be such a big number?

Reply:
Added comment